zsh_completions.go

```diff
@@ -0,0 +1,123 @@
+package cobra
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+)
+
+type fWriter struct {
+	io.Writer
+}
+
+func (fw *fWriter) fWriteLn(format string, a ...interface{}) (int, error) {
+	return io.WriteString(fw, fmt.Sprintf(format+"\n", a...))
```

Comment:
I don't like this fWriter struct.
Why don't you use just `fmt.Fprintln(w, format, a1, a2)` like in bash_completions.go? It's simpler and nicer